viite-backend/database/src/main/scala/fi/vaylavirasto/viite/dao/RoadNameScalikeDAO.scala

```diff
@@ -25,6 +25,59 @@ object RoadNameScalikeDAO extends ScalikeJDBCBaseDAO {
   private val rn = RoadNameScalike.syntax("rn") // rn is an alias for RoadNameScalike object to be used in queries
   private val rw = RoadwayScalike.syntax("rw")
 
+  // Base query to select all columns from ROAD_NAME table
+  private val selectAllFromRoadNameQuery = sqls"""
+  SELECT ${rn.id}, ${rn.roadNumber}, ${rn.roadName}, ${rn.startDate}, ${rn.endDate},
+         ${rn.validFrom}, ${rn.validTo}, ${rn.createdBy}
+  FROM ${RoadNameScalike.as(rn)}
+"""
+
+  // Method to run select queries for RoadName objects
+  private def queryList(query: SQLSyntax): Seq[RoadName] = {
+    runSelectQuery(sql"$query".map(RoadNameScalike.apply))
+  }
+
+  /**
+   * Get all road names from ROAD_NAME table with road number and optional start and end dates
+   *
+   * @param startDate Optional start date for the road names
+   * @param endDate   Optional end date for the road names
```

Comment:
Heeetkone, voiko startDatekin olla null?!?
Eeeei kyl pitäis olla koskaan.
endDate varmaan on null, jos nimi on vielä voimassa.

Reply:
Eiku siis. Otetaas takas. Kommenttieja tarttee säätää.
Jotain tähän tyyliin:

   * @param startDate Optional oldest date from where the road names are returned
   * @param endDate   Optional latest date from where the road names are returned

Mut en oo tohonkaan tyytyväinen, mielellään vielä paremmin/selkeämmin 😅 
Nyt noista kommenteista saa sellaisen käsityksen, että niillä rajataan tulokset just tasan noihin päiviin.